docs/examples/queued_io.rst

```diff
@@ -5,13 +5,21 @@ This example uses :any:`nixnet.session.FrameInQueuedSession` and
 :any:`nixnet.session.FrameOutQueuedSession` to demonstrate how queued sessions
 work.
 
+CAN Queued I/O
+---------------
+
+.. literalinclude:: ../../nixnet_examples/can_frame_queued_io.py
+
 To adapt this example to LIN frames, reference signals in a database that use LIN:
```

Comment:
Personally, I'd recommend making the new rst file be about adapting CAN examples for LIN and include all of this text.